src/SourceBuild/content/eng/pipelines/vmr-license-scan.yml

```diff
@@ -93,11 +93,10 @@ jobs:
       -bl:$(Build.SourcesDirectory)/artifacts/log/Debug/BuildTests_$(date +"%m%d%H%M%S").binlog 
       -flp:LogFile=$(Build.SourcesDirectory)/artifacts/logs/BuildTests_$(date +"%m%d%H%M%S").log
       -clp:v=m
-      -e SMOKE_TESTS_LICENSE_SCAN_PATH=$(repoPath)
```

Comment:
Can you also do a test run of these changes on the [license scan pipeline](https://dev.azure.com/dnceng/internal/_build?definitionId=1301)? This pipeline can be run after these changes are synced to a branch of the internal VMR.